js-code-style.md

```diff
@@ -1,9 +1,20 @@
 # Javascript Code Style
 
-## eslint with the `eslint-config-airbnb-base` preset, `eslint-config-airbnb` for React projects;
+## ESLint with the `eslint-config-airbnb-base` preset, `eslint-config-airbnb` for React projects;
 
 We can start a project to extend one of those bases if we find we have issues with some rules.
 
+### For tests
```

Comment:
I really think that having a standard test output format (TAP) is much better than a custom reporter. This allows anyone to format test output as the developer wishes.